packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1861,6 +1895,19 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             type: envelope.contents.type,
         };
 
+        // Only collect signal telemetry for messages sent by the current client.
+        if (message.clientId === this.clientId && this.connected) {
+            // Check to see if the signal was lost.
+            if (this._perfSignalData.trackingSignalSequenceNumber !== undefined &&
+                envelope.clientSignalSequenceNumber > this._perfSignalData.trackingSignalSequenceNumber) {
+                this._perfSignalData.signalsLost++;
+                this._perfSignalData.trackingSignalSequenceNumber = undefined;
+            } else if (envelope.clientSignalSequenceNumber % this.defaultTelemetrySignalSampleCount === 1) {
```

Comment:
This should be comparison
envelope.clientSignalSequenceNumber === this._perfSignalData.trackingSignalSequenceNumber